kernelci.org/content/en/docs/legacy/instances/chromeos/chromeos_changelog.md

```diff
@@ -24,6 +24,63 @@ For an up-to-date overview of current and planned releases, please visit the [sc
 
```

Comment:
This should be `R118` (see line above as GH doesn't allow me to comment directly on it)